apps/api/internal/update/dispatch.go

```
@@ -0,0 +1,373 @@
+//
+// Two hours is short enough that no plausible business-hours window is crossed
+// and long enough that an ordinary deploy or a brief restart does not silently
+// destroy an operator's scheduled work.
+const dispatchGraceWindow = 2 * time.Hour
+
+// dueRunScanLimit bounds how many due runs one dispatcher pass handles, so an
+// unbounded backlog cannot make a single tick unbounded. The remainder is
+// picked up by the next pass; ListDueUpdateRuns orders by scheduled_at ASC, so
+// a backlog drains in the order the operators asked for. Mirrors
+// staleTaskReapLimit's role for the reaper.
+const dueRunScanLimit = 200
```

golangci-lint (2.12.2)

[error] const dueRunScanLimit is unused

(unused)